demo/AppDelegate.swift

```diff
@@ -36,7 +41,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         containerController.visionViewController = VisionPresentationViewController()
         
-        let menuViewController = MenuViewController()
+        let menuViewController = MenuViewController(tosLink: Constants.tosLink, privacyPolicyLink: Constants.privacyPolicyLink)
```

Comment:
it seems that parametrizing a component with global constants creates more code than benefit. It definitely makes sense to use DI for dynamic content but may be an overkill for static strings.

Reply:
done